app/controllers/application_controller.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationController < ActionController::Base
+end
```

Comment:
- [ ] [OPTIONAL] It can be easy for you if you created an API-based app. A normal rails app can be a bit difficult to set up.